services/autoscaling/src/simcore_service_autoscaling/core/settings.py

```diff
@@ -240,6 +240,11 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
         ],
         description="Enables local development log format. WARNING: make sure it is disabled if you want to have structured logs!",
     )
+    AUTOSCALING_LOG_FILTER_MAPPING: dict = Field(
```

Comment:
what about doing some validation for `LOG_FILTER_MAPPING` ?? 
> is a dictionary that maps specific loggers (such as 'uvicorn.access' or 'gunicorn.access') to a list of log message patterns that should be filtered out.

You can define the validator once and reuse it in all settings classes